README.md

```diff
@@ -54,7 +54,7 @@ If you originally installed Mixpanel via Bower at its previous home ([https://gi
 
 ## Running tests
 - Install development dependencies: `npm install`
-- Start test server: `npm start`
+- Start test server: `npm test`
```

Comment:
I seemed to have to run `npm test` and not `npm start` like the documentation stated.